samples/cellular/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -367,6 +368,171 @@ To monitor the GNSS module (for instance, to see whether A-GPS or P-GPS assistan
+If an attacker obtains the private key (``cred_my_wifi_dev_prv.pem``), they will be able to impersonate your device.
+Since the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` does not protect this private key from extraction, it should not be used in production!
+
+Next, create a file named ``my_wifi_dev_cred.csv`` and give it the following contents:
+
+.. code-block:: csv
+
+   XXXX,,,,"YYYYYYYY"
+
+Where XXXX is replaced by your chosen device ID, and YYYYYYYY is replaced by the exact contents of ``cred_my_wifi_dev_crt.pem``.
+
+For example:
+
+.. code-block:: csv
+
+   my_wifi_dev,,,,"-----BEGIN CERTIFICATE-----
+   sCC8AtbNQhzbp4y01FEzXaf5Fko3Qdq0o5LbuNpVA7S6AKAkjt17QzKJAiGWHakh
+   RnwzoA2dF4wR0rMP5vR6dqBblaGAA5hN7GE2vPBHTDNZGJ6tZ9dnO6446dg9gGds
+   eeadE1HdVnUj8nb+7CGm39vJ4fuNk9vogH0nMdxjCnXAinoOMRx8EklQsR747+Gz
+   sxcdVYuNEb/E2vWBHTDNZGJ6tZC1JC9d6/RC3Vb1JC4tWnK9mk/Jw984ZuYugpMc
+   1t9umoGFYCz0nMdxjCnXAbnoOMC5A0RxcWPzxfC5A0RH+j+mwoMxwhgfFY4EhVxp
+   oCC8labNQhzRC3Vc1JC4tWnK9mpVA7k/o5LbuNpVA7S6AKAkjt17QzKJAiGWHakh
+   RXwcoAndF4wPzxfC5A0RHponmwBHTDoM7GE2vPBHTDNZGJ6tZ9dnO6446dg9gGds
+   eefdE1HcVnULbuNpVA7S6AKAkjxjCnt1gH0nMdxjCnXAinoOMRx8EklQsR747+Fz
+   srm/VYaNEb/E2vPBHTDNZGJ6tZc1JC9d6/RC3Vc1JC4tWnK9mk/Jr984ZuYugpMc
+   nt9uZTGFYCzZD0FFAA5NAC4i1PARStFycWPzxfC5A0RqodhswoMxwhgfFY4EhVx=
+   -----END CERTIFICATE-----
+   "
+
+Do not attempt to use this example verbatim, it has been filled with dummy data.
+
+Next, navigate to the `Provision Devices <nRF Cloud Portal Provision Devices_>`_ page of the nRF Cloud portal, drag and drop ``my_wifi_dev_cred.csv``, and press ``Upload and Provision``.
+You should see a message stating that the file was uploaded successfully, and a device with your chosen device ID should appear in the `Devices <nRF Cloud Portal Devices_>`_ page.
+
+Next, download `Amazon Root CA 1`_ and save it as ``AmazonRootCA1.pem``.
+See `CA certificates for server authentication in AWS IoT Core`_ for more details.
+Do not confuse this CA cert with your self-managed CA cert.
+
+Next, wrap ``cred_my_wifi_dev_crt.pem``, ``cred_my_wifi_dev_prv.pem``, and ``AmazonRootCA1.pem`` in quotes, and save them as ``client-cert.pem``, ``private-key.pem`` and ``ca-cert.pem`` respectively.
+
+. tabs::
+
+   .. group-tab:: Bash
+
+      .. code-block:: console
+
+        awk 'NF { print "\""$0"\\n\""}' cred_my_wifi_dev_crt.pem > client-cert.pem
+        awk 'NF { print "\""$0"\\n\""}' cred_my_wifi_dev_prv.pem > private-key.pem
+        awk 'NF { print "\""$0"\\n\""}' AmazonrootCA1.pem > ca-cert.pem
+
+   .. group-tab:: PowerShell
+
+      .. code-block:: console
+         gc .\cred_my_wifi_dev_crt.pem | %{"""$_\n"""} | out-file -encoding utf8 client-cert.pem
+         gc .\cred_my_wifi_dev_prv.pem | %{"""$_\n"""} | out-file -encoding utf8 private-key.pem
+         gc .\AmazonrootCA1.pem | %{"""$_\n"""} | out-file -encoding utf8 ca-cert.pem
+
+If this succeeds, you should now have the following three files:
```

Comment:
Is this likely to fail?

Reply:
Only if the user makes a mistake. I include phrases like this just to give the user a way of checking their own work

I can rephrase as just `you should now have the following three files`